pages/index.tsx

```diff
@@ -5,6 +5,10 @@ import Image from "next/image";
 import JSConfetti from "js-confetti";
 import { useRef, useEffect, use } from "react";
 
+let images = [
```

Comment:
later, lets excise this into another file

Reply:
agreed, another thing to do is make the file names not.... horrid

Reply:
i love index